README.md

```diff
@@ -1,24 +1,154 @@
 
-* Database creation
+## 🛠 Built With <a name="built-with"></a>
 
-* Database initialization
+### Tech Stack <a name="tech-stack"></a>
 
-* How to run the test suite
+- <a href="https://www.ruby-lang.org/es/">ROR</a>
+- <a href="https://www.postgresql.org/">Postgresql</a>
+- <a href="https://rspec.info/">Rspec</a>
 
-* Services (job queues, cache servers, search engines, etc.)
+<!-- Features -->
 
-* Deployment instructions
+### Key Features <a name="key-features"></a>
 
-* ...
+- **[Users combine rails and react in one application]**
+- **[Users consume API]**
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need the following installed in your local system
+
+<ul>
+<li>Ruby</li>
+<li>Git CLI</li>
+<li>Github account</li>
+</ul>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd hello-rails-react
+  git clone https://github.com/flemton/hello-rails-react.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd hello-rails-react
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+``` sh
+rails s
+```
+### Run tests
+
+-  Type `Rspec`: To run test `
+-  Type `npx stylelint "**/*.{css,scss}"` to run for stylelint errors
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Alhassan (Newton) Osman**
+
+- GitHub: [@flemton](https://github.com/flemton)
+- Twitter: [@oalhassan847](https://twitter.com/oalhassan847)
+- LinkedIn: [Newtongh](https://www.linkedin.com/in/newtongh/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Style components]**
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/flemton/hello-rails-react/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please leave a STAR to the [Github repo](https://github.com/flemton/
+hello-rails-react/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to appreciate [Microverse](https://www.microverse.org/) for providing this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] [REQUIRED] Kindly fix the broken MIT link by adding the `LICENSE` file to the root directory. This is a hard project requirement.
<img width="1135" alt="Screenshot 2023-12-10 at 04 15 51" src="https://github.com/flemton/hello-rails-react/assets/58692590/d0187bc4-de82-4cd6-966a-f7be394db8e7">

Reply:
well resolved! ✔️